docs/getting-started/installation.md

```diff
@@ -70,14 +70,15 @@ centrifugo --config=config.json
 
 Centrifugo server has a docker image [available on Docker Hub](https://hub.docker.com/r/centrifugo/centrifugo/).
 
+Generate a configuration file.
 ```
-docker pull centrifugo/centrifugo
```

Comment:
Explicitly pulling an image before run is not needed.

Reply:
This allows updating an image, otherwise `docker run` launches locally cached image version - which was an issue several times. Let's remove for now, but I'll try to find some way to mention this.

Reply:
This allows updating image, otherwise docker runs locally cached image - this was an issue a couple of times. Let's remove for now, will try to find some way to mention this